src/index.js

```diff
@@ -1,6 +1,7 @@
 const p = require('path')
 const babylon = require('babylon')
 const requireFromString = require('require-from-string')
+const babel = require('babel-core')
```

Comment:
Is this in our dependencies? Note also that the argument to our plugin function is actually babel-core so you could probably just use that.